### DISCUSSION TITLE
能否增加一个C#版本

### DISCUSSION THREAD

**Discussion by sangeren** [RapidAI/RapidLatexOCR#31] [Ideas]
谢谢

**Comment by SWHL** [comment]
不会呀，欢迎一起写啊

**Reply by sangeren** [reply]
可以的，不过我不清楚推理的具体过程，比如图片应该怎么预处理？怎么解码字符串？
有没有相关的文档 ，^_^

**Reply by SWHL** [reply]
这个没有，只有源码可以参考了。可以借助GPT试试

**Reply by sangeren** [reply]
我大概写好了，正常可以使用。

**Comment by SWHL** [comment]
溜啊，欢迎贡献代码哈

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年12月17日 18:38 |
| 收件人 | RapidAI/RapidLaTeXOCR ***@***.***> |
| 抄送人 | SWHL ***@***.***>,
State ***@***.***> |
| 主题 | Re: [RapidAI/RapidLaTeXOCR] 能否增加一个C#版本 (Discussion #31) |

我大概写好了，正常可以使用。
